### PR TITLE
api created

### PR DESCRIPTION
**Pull request by nimplay**
- [ ] Delete previous code
- [ ] add the asyncronous JS
- [ ] call the api
- [ ] programming submit buttom
- [ ] programming refresh buttom

### CODE REVIEW

File: src/modules/refresh.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+const refresh = async () => {
+  const listUl = document.querySelector('.score-table');
+  const results = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/bmBsWnBXvVGZ5GV7fqzY/scores/');
+  const data = await results.json();
+  const arr = data.result;
+  arr.forEach((e) => {
+    const savedRecord = Result.record(e.user, e.score);
+    listUl.innerHTML += savedRecord;
+  });
+};
```
Comment (AnselemOdims) [inline src/modules/refresh.js]: - [ ] At the moment, anytime the refresh button is clicked it repopulates the list with new entries. Please kindly ensure it only shows the list of item and not repeat them.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @nimplay, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by nimplay** [comment]
All the requirements of the previous reviewer were corrected

**Review by smunozmo — APPROVED** [review]
Hi @nimplay, 👋 

## Highlights 👏 

- Your app is working properly, nice job! 😎 
- Well done using arrow functions!
- Excelent use of `await` and `fetch`, great!
- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

# APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**